### ISSUE TITLE
"An error happened during database sync for artworks: undefined"

### ISSUE THREAD

**Issue by almet** [compactd/compactd#62]
I'm currently trying to setup the project on my dev machine to work on it and solve the integration problem with some gazelle trackers, but I encounter this error.

Any idea on how I can circumvent this? Thanks!

**Comment by almet** [comment]
After checking the couchdb logs, it seems that I'm getting a few `401`, not sure why thought.

`[notice] 2018-04-04T19:52:24.328243Z couchdb@127.0.0.1 <0.15484.0> f7c3dec8be localhost:9000 127.0.0.1 undefined PUT /artworks/?_nonce=1522871544320 401 ok 1`

**Comment by almet** [comment]
I wonder if this could be related to the fact that my `Music` folder was actually empty, so it might not have initiated the `artworks` collection.

**Comment by almet** [comment]
Yes, it seems to be the reason!

**Comment by vinz243** [comment]
Nice thanks  for the help ! did you solved your issue then?

**Comment by almet** [comment]
Yes, I just had to add an album to my music folder and issue `compactd --reset` and that was gone, but I think it would be better to have an error message that would lead the user to do this instead of this weird error.

**Comment by vinz243** [comment]
I agree, need to solve those errors for 2.0
